src/app/cases/page.tsx

```diff
@@ -94,6 +100,26 @@ export default function Page() {
               setCaseFilters({ ...caseFilters, countries: v as string[] })
             }
           />
+          <AuthButtons>
```

Comment:
can we scoot the buttons over to the left a bit so their end aligns with the interest form component's end? see:

figma
<img width="254" alt="Screenshot 2023-11-23 at 8 44 04 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/905f7b46-e59e-4812-8b11-d53a33620a44">

vs.

app

<img width="198" alt="Screenshot 2023-11-23 at 8 44 23 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/cb62237a-1983-4105-ab31-abec825eaa78">

Reply:
also while we're at it...would you be down to add the drop shadow (?) underneath the title/filters in this PR? 

<img width="819" alt="Screenshot 2023-11-23 at 8 52 26 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/85601c4f-5d77-4f7d-ac2b-ca71f0d5b061">

not sure how difficult this would be so lmk if you'd rather i separate this into a different sprint. but yeah since this pr is on the shorter side, thought it might be good to toss this in as well!

Reply:
oh missed the second part
will do 🫡